network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -87,9 +87,10 @@ <h1 class="h1-heading text-center mt-0">{% trans "Be Smart. Shop Safe." %}</h1>
               <img
                 class="product-thumbnail"
                 {% if USE_CLOUDINARY %}
-                  src="{% cloudinary_url product.cloudinary_image quality=50 fetch_format="auto" crop="fit" width=600 %}"
+                  src="{% cloudinary_url product.cloudinary_image quality=50 fetch_format="auto" crop="fit" width=200 %}"
```

Comment:
Oh yeah I noticed this was set to 600, which seemed HUGE for a tiny thumbnail image. If this is not acceptable I can revert this.

Reply:
Opened #5677 and will revert this back to 600 👍